tests/ast/nodes/test_get_descendants.py

```diff
@@ -0,0 +1,73 @@
+from pathlib import (
```

Comment:
Ditto this. Why test it?

Reply:
In returning descendants, the sequence is important. These tests confirm that nodes are always returned sorted by source offset and depth.